### PR TITLE
Manage State Or Provinces

### PR DESCRIPTION
**Pull request by DieuNguyenThi**
Hi everyone,

Plz help me to review this PR,

- Config absolute path
- Refactor code
- Manage state or province: Page list, Create, Update, Filter, Validate, Paging...

Thanks!

### CODE REVIEW

File: backoffice/modules/system/components/CountryGeneralInformation.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { FieldErrorsImpl, UseFormRegister, UseFormSetValue, UseFormTrigger } from 'react-hook-form';
-import { CheckBox } from '../../../common/items/Input';
-import { Input } from '../../../common/items/Input';
+import { CheckBox } from 'common/items/Input';
+import { Input } from 'common/items/Input';
```
Comment (duccaominh) [inline backoffice/modules/system/components/CountryGeneralInformation.tsx:3]: good approach 🤗

File: location/src/main/java/com/yas/location/controller/CountryController.java [hunk 2]
```diff
@@ -34,59 +35,59 @@ public CountryController(CountryRepository countryRepository, CountryService cou
...
-    @GetMapping({"/backoffice/countries/paging"})
+    @GetMapping({"/paging"})
     public ResponseEntity<CountryListGetVm> getPageableCountries(@RequestParam(value = "pageNo", defaultValue = Constants.PageableConstant.DEFAULT_PAGE_NUMBER, required = false) int pageNo,
                                                                  @RequestParam(value = "pageSize", defaultValue = Constants.PageableConstant.DEFAULT_PAGE_SIZE, required = false) int pageSize) {
 
         return ResponseEntity.ok(countryService.getPageableCountries(pageNo, pageSize));
     }
 
-    @GetMapping({"/backoffice/countries"})
+    @GetMapping({""})
```
Comment (duccaominh) [inline location/src/main/java/com/yas/location/controller/CountryController.java:45]:
```Java
@GetMapping
```

File: location/src/main/java/com/yas/location/service/StateOrProvinceService.java [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+            throw new DuplicatedException(Constants.ERROR_CODE.NAME_ALREADY_EXITED, stateOrProvincePostVm.name());
+        }
+        Country country = countryRepository.findById(stateOrProvincePostVm.countryId()).orElseThrow(()
+                -> new NotFoundException(Constants.ERROR_CODE.STATE_OR_PROVINCE_NOT_FOUND, stateOrProvincePostVm.countryId()));
+
+        StateOrProvince stateOrProvince = StateOrProvince.builder()
+                .name(stateOrProvincePostVm.name())
+                .code(stateOrProvincePostVm.code())
+                .code(stateOrProvincePostVm.type())
+                .country(country)
```
Comment (duccaominh) [inline location/src/main/java/com/yas/location/service/StateOrProvinceService.java:52]: Do we really need to load the whole country entity here

**Review by duccaominh — CHANGES_REQUESTED** [review]
hi @DieuNguyenThi  please review the refactor code on backend side for me

**Review by chanhnguyen6975 — APPROVED** [review]

**Review by duccaominh — APPROVED** [review]

**Pull request merged**